modules/Kconfig.tls-generic

```diff
@@ -37,6 +37,9 @@ config MBEDTLS_SSL_EXPORT_KEYS
 	bool "Enable support for exporting SSL key block and master secret"
 	depends on MBEDTLS_TLS_VERSION_1_0 || MBEDTLS_TLS_VERSION_1_1 || MBEDTLS_TLS_VERSION_1_2
 
+config MBEDTLS_SSL_ALPN
+    bool "Enable support for setting the supported Application Layer Protocols"
```

Comment:
How about adding the dependency to TLS, just as in line 38?

Reply:
I am a total TLS newbie, is this extension something that is applicable to all 3 TLS versions listed on line 38?

Reply:
Hmm I haven't found any TLS version limitation in https://tools.ietf.org/html/rfc7301, so I'd assume it can be used with either.